### PR TITLE
Updated references and fixed typos

### PR DESCRIPTION
**Pull request by jsagardoy**
Updated file due to issue #126

### CODE REVIEW

File: version 4/00 intro/05 jquery/README.md [hunk 1]
```diff
@@ -5,29 +5,30 @@ basic pillars of web development, consuming third party libraries.
 
 In this demo we will install a legacy library (jquery) via npm, define it as global, and use it. Finally we will end up creating a separate bundle for libraries.
 
-We will start from sample _00 Intro/03 Output_.
+We will start from sample _00 intro/_05 jquery_.
```
Comment (brauliodiez) [inline version 4/00 intro/05 jquery/README.md]: I think we take as starting point 03 Output sample then we code it step by step and we get 05 Jquery?

File: version 4/00 intro/05 jquery/README.md [hunk 2]
```diff
@@ -5,29 +5,30 @@ basic pillars of web development, consuming third party libraries.
...
 
 Summary steps:
  - Install jquery via npm.
- - Setup a global alias ($).
- - Create some sample code using this library.
- - Break into two bundles `app.js` and `vendor.js`.  
+ - Steps to buid it:
+  - Prerequisites
+  - Steps
+  - Running and testing it
```
Comment (brauliodiez) [inline version 4/00 intro/05 jquery/README.md]: In the steps we summarize the detailed steps, so a developer can give  a read and check what he is going to do (e.g. you have to install this pacakge, setup this global variable and break into two bundles).

File: version 4/00 intro/05 jquery/README.md [hunk 3]
```diff
@@ -5,29 +5,30 @@ basic pillars of web development, consuming third party libraries.
...
 
 # Steps to build it
 
 ## Prerequisites
 
-Prerequisites, you will need to have nodejs installed (at least v 8.9.2) on your computer. If you want to follow this guide steps you will need to take as starting point sample _04 Output_.
+Prerequisites, you will need to have nodejs installed (at least v 8.9.2) on your computer. If you want to follow this guide steps you will need to take as starting point sample _00 intro/_05 jquery_.
```
Comment (brauliodiez) [inline version 4/00 intro/05 jquery/README.md]: I think is 03 output

File: version 4/00 intro/05 jquery/README.md [hunk 4]
```diff
@@ -5,29 +5,30 @@ basic pillars of web development, consuming third party libraries.
...
 
 - `npm install` to install previous sample packages:
 
 ```
 npm install
 ```
-- Let's start by downloading the jquery library via npm. In this case we will execute the following command on the command prompt ```npm install jquery --save```.
-**note down**: this time we are not adding the `-d` suffix to the parameter, this time the jquery package is a dependency of the web app not of the build process.
+- Let's start by downloading the jquery library via npm. In this case we will execute the following command on the command prompt: ```npm install jquery --save```.
+**Note down**: this time we are not adding the `-d` suffix to the parameter, this time the jquery package is a dependency of the web app not of the build process.
```
Comment (brauliodiez) [inline version 4/00 intro/05 jquery/README.md]: For the notes we can use ">"

> Note down: this time...

Is a special character from markdown

**Comment by jsagardoy** [comment]
File updated with the comments from @brauliodiez

**Comment by brauliodiez** [comment]
Good stuff,reviewed and merging it

**Pull request merged**